src/madness/world/type_traits.h

```diff
@@ -61,7 +61,12 @@ namespace madness {
     /// This defines stuff that is serialiable by default rules ... basically anything contiguous
     template <typename T>
     struct is_serializable {
-        static const bool value = std::is_fundamental<T>::value || std::is_member_function_pointer<T>::value || std::is_function<T>::value  || std::is_function<typename std::remove_pointer<T>::type>::value || std::is_pod<T>::value;
+        static const bool value = \
+            std::is_fundamental<T>::value || \
+            std::is_member_function_pointer<T>::value || \
+            std::is_function<T>::value  || \
+            std::is_function<typename std::remove_pointer<T>::type>::value || \
+            (std::is_pod<T>::value && !std::is_pointer<T>::value);
```

Comment:
- per http://en.cppreference.com/w/cpp/types/is_pod std::is_pod will be deprecated
- I think this clause is to make trivially-copyable composite types built of serializable elements to be serializable. There is no way right now (without reflection) to properly check this. My suggestion is to say here `(std::is_class<T>::value || std::is_array<T>::value) && std::is_trivially_copyable<T>::value` to make intent as explicit as possible.

The shortcomings of `std::is_trivially_copyable` are discussed https://groups.google.com/a/isocpp.org/forum/#!topic/std-discussion/wphImiqfX7Y%5B1-25%5D. It's a loaded topic.

Reply:
OK.  U R suggesting the following ??

+        static const bool value = \
+            std::is_fundamental<T>::value || \
+            std::is_member_function_pointer<T>::value || \
+            std::is_function<T>::value  || \
+            std::is_function<typename std::remove_pointer<T>::type>::value || \
+            ((std::is_class<T>::value || std::is_array<T>::value) && std::is_trivially_copyable<T>::value);

Reply:
```
static const bool value = \
std::is_arithmetic<T>::value || \
std::is_member_function_pointer<T>::value || \
std::is_function<T>::value  || \
std::is_function<typename std::remove_pointer<T>::type>::value || \
((std::is_class<T>::value || std::is_array<T>::value) && std::is_trivially_copyable<T>::value);
```